### ISSUE TITLE
SA1120: CommentsMustContainText

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#53]
http://www.stylecop.com/docs/SA1120.html

**Comment by johnkoerner** [comment]
Grabbed

**Comment by johnkoerner** [comment]
Thoughts on if the following should raise the warning:

    ////
    Console.WriteLine("a")

I tried with the StyleCop.MsBuild nuget package and it does not raise the warning on that line.  To me, it is still a empty comment, so I would be in favor of raising the warning on it.

**Comment by sharwell** [comment]
It should not. 3 slashes is a documentation comment, and 4 (or more) is the special syntax allowed for commenting blocks of code.

**Comment by johnkoerner** [comment]
I have a working version, but one interesting report I got on a brand new console app was in the AssemblyInfo.cs file.  

```
// Version information for an assembly consists of the following four values:
//
//      Major Version
//      Minor Version 
//      Build Number
//      Revision
//
// You can specify all the values or you can default the Build and Revision Numbers 
// by using the '*' as shown below:
// [assembly: AssemblyVersion("1.0.*")]
[assembly: AssemblyVersion("1.0.0.0")]
```

It's reporting on the two blank lines in that comment block. Is that acceptable, or should we try and be smart about the blank line being in a block of other single line comments?

**Comment by sharwell** [comment]
@johnkoerner Thanks for testing this thoroughly. This is a common case; I'm glad you caught it. :smile:
